pmsoService/product_flow/features/import_request/api/views/delete_update_views.py

```
@@ -0,0 +1,48 @@
+    serializer_class = RequestImportUpdateSerializer
+    lookup_field = 'id'
+
+    def get_queryset(self):
+        return ProductImportInfo.objects.filter(product_order__isnull=True)
+
+    def get_serializer_context(self):
+        context = super().get_serializer_context()
+        context["import_request"] = self.get_object()
+        return context
+
+    def patch(self, request, *args, **kwargs):
```

Ruff (0.13.3)

Unused method argument: `args`

(ARG002)